### PR TITLE
Support for Inovelli VZW31-SN & cleanup 

### PR DESCRIPTION
**Pull request by InovelliUSA**

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -11,8 +11,13 @@ const libColor = require('../lib/color');
 const exposes = require('../lib/exposes');
 
 const manufacturerOptions = {
-    sunricher: {manufacturerCode: herdsman.Zcl.ManufacturerCode.SHENZHEN_SUNRICH},
-    xiaomi: {manufacturerCode: herdsman.Zcl.ManufacturerCode.LUMI_UNITED_TECH, disableDefaultResponse: true},
+    sunricher: {
```
[the remark this answers is not in the document]
Comment (nathanfiscus) [reply converters/toZigbee.js]: Oops! Sorry about that, I did not mean to do that.  I've reverted the formatting.

File: converters/toZigbee.js [hunk 2]
```diff
@@ -7409,6 +7409,63 @@ const converters = {
             }
         },
     },
+    inovelli_led_effect: {
```
Comment (Koenkk) [inline converters/toZigbee.js:7412]: Could you also move these to the `tzLocal` of `inovelli.js`?

Comment (nathanfiscus) [reply converters/toZigbee.js]: Done.

File: devices/inovelli.js [hunk 3]
```diff
@@ -0,0 +1,1223 @@
...
+                .withDescription('Individual LED to target.'),
+        )
+        .withFeature(
+            exposes
+                .enum('effect', ea.SET_STATE, [
+                    'Off',
+                    'Solid',
+                    'Fast Blink',
+                    'Slow Blink',
+                    'Pulse',
```
Comment (Koenkk) [inline devices/inovelli.js:1099]: Please also make these snake case

File: devices/inovelli.js [hunk 4]
```diff
@@ -0,0 +1,1223 @@
...
+                .enum('effect', ea.SET_STATE, [
+                    'Off',
+                    'Solid',
+                    'Chase',
+                    'Fast Blink',
+                    'Slow Blink',
+                    'Pulse',
+                    'Open/Close',
+                    'Small to Big',
+                    'Clear Effect',
```
[the remark this answers is not in the document]
Comment (nathanfiscus) [reply devices/inovelli.js]: Done.

File: devices/inovelli.js (hunk 3 above)
Comment (nathanfiscus) [reply devices/inovelli.js]: Done.

**Pull request merged**